canonical-web-and-design/anbox-cloud.io
# Creates testing vertical page

## Done

- Creates testing vertical page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/testing
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [Copy-docs](https://docs.google.com/document/d/1WpN0_o092-yy37SKYiddLpJ0QVdSBseAGCyV_5qWeTk/edit#)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4010
